src/app/rosetta/lib/account.ml

```diff
@@ -31,25 +31,8 @@ end
 
 module Sql = struct
   module Balance_from_last_relevant_command = struct
-    let query =
-      Caqti_request.find_opt
-        Caqti_type.(tup2 string int64)
-        Caqti_type.(tup2 int64 int64)
-        {sql|
-WITH RECURSIVE chain AS (
-  (SELECT id, state_hash, parent_id, height, global_slot_since_genesis, timestamp
-  FROM blocks b
-  WHERE height = (select MAX(height) from blocks)
-  ORDER BY timestamp ASC
-  LIMIT 1)
-
-  UNION ALL
-
-  SELECT b.id, b.state_hash, b.parent_id, b.height, b.global_slot_since_genesis, b.timestamp FROM blocks b
-  INNER JOIN chain
-  ON b.id = chain.parent_id AND chain.id <> chain.parent_id
-),
-
+    let common_sql =
```

Comment:
a comment here how this is shared by `query_old` and `query_recent` would be helpful for future audiences